{
    "CVE_data_meta": {
        "ASSIGNER": "cve@mitre.org",
        "ID": "CVE-2024-27351",
        "STATE": "PUBLIC"
    },
    "affects": {
        "vendor": {
            "vendor_data": [
                {
                    "product": {
                        "product_data": [
                            {
                                "product_name": "n/a",
                                "version": {
                                    "version_data": [
                                        {
                                            "version_value": "n/a"
                                        }
                                    ]
                                }
                            }
                        ]
                    },
                    "vendor_name": "n/a"
                }
            ]
        }
    },
    "data_format": "MITRE",
    "data_type": "CVE",
    "data_version": "4.0",
    "description": {
        "description_data": [
            {
                "lang": "eng",
                "value": "In Django 3.2 before 3.2.25, 4.2 before 4.2.11, and 5.0 before 5.0.3, the django.utils.text.Truncator.words() method (with html=True) and the truncatewords_html template filter are subject to a potential regular expression denial-of-service attack via a crafted string. NOTE: this issue exists because of an incomplete fix for CVE-2019-14232 and CVE-2023-43665."
            }
        ]
    },
    "problemtype": {
        "problemtype_data": [
            {
                "description": [
                    {
                        "lang": "eng",
                        "value": "n/a"
                    }
                ]
            }
        ]
    },
    "references": {
        "reference_data": [
            {
                "url": "https://groups.google.com/forum/#!forum/django-announce",
                "refsource": "MISC",
                "name": "https://groups.google.com/forum/#!forum/django-announce"
            },
            {
                "url": "https://docs.djangoproject.com/en/5.0/releases/security/",
                "refsource": "MISC",
                "name": "https://docs.djangoproject.com/en/5.0/releases/security/"
            },
            {
                "refsource": "CONFIRM",
                "name": "https://www.djangoproject.com/weblog/2024/mar/04/security-releases/",
                "url": "https://www.djangoproject.com/weblog/2024/mar/04/security-releases/"
            }
        ]
    }
}